app/src/reducers/index.js

```diff
@@ -111,8 +112,9 @@ const rootReducer = (state={}, action) => {
     });
     updatedSections[sectionIndex] = {
       ...updatedSections[sectionIndex],
-      sequenceSlots: [...updatedSections[sectionIndex].sequenceSlots.splice(action.slotIndex, 1)]
+      sequenceSlots: [...updatedSections[sectionIndex].sequenceSlots]
```

Comment:
this was a bad idea, I was spreading the result of splice, which was an array with the sliced parts.. no bueno